db-engine/src/main/java/com/splicemachine/db/catalog/SystemProcedures.java

```diff
@@ -1710,13 +1718,19 @@ private static boolean hasTable(Connection conn,String schemaName,
             // If there is already a schema then we will let the admin decide what he want to do
             // like granting privileges to that schema manually. Look at SPLICE-1030
             addSchema(userName, userName, lcc);
+            status = true;
 
         }catch(StandardException se){
             throw PublicAPI.wrapStandardException(se);
+        }catch (SQLException sqle){
+            status = false;
+            reason = sqle.getMessage();
+            throw sqle;
+        }
+        finally {
+            if (AUDITLOG.isInfoEnabled())
+                AUDITLOG.info(StringUtils.logSpliceAuditEvent(currentUser,AuditEventType.CREATE_USER.name(),status,ip,lcc.getStatementContext().getStatementText(),reason));
```

Comment:
By passing down the statement directly, we log the password directly in the splice-audit log. Is it appropriate? Same concern for the logging of the other two system procedures.

Reply:
We definitely don't want to include the password in the log, even if it's the failed password.

Reply:
That is the concern. Even though the splice-audit.log is permission strict, then sensitive info should not be exposed. One solution is to change the log4j configuration using MaskPatternLayout to hide sensitive info. Another choice is doing MaskPattern in internal to avoid the filtering regular expression is too complicated to customer. 
I prefer the second one.